site/content/integrate/apps/_index.md

```diff
@@ -31,6 +31,22 @@ Read the [JavaScript start guide]({{< ref "quick-start-js" >}}) or [Go quick sta
 
 ## FAQ
 
+### When would you build an app vs a custom slash command vs a webhook vs a plugin?
+
+That depends on your use case, as they each have benefits. Apps can be advertised, shared, and installed by others via the Marketplace. Comparatively, a custom slash command requires some configuration and is less interactive.
+
+The built-in incoming webhook is rigid and requires specific fields to be posted to it's URL, but some external systems don't allow the customization of the fields included in the webhook payload. Compared to built-in webhooks, An App can be customized to receive an HTTP webhook posting from another system, and can process the incoming data then augment it or make an actionable message in a channel.
```

Comment:
2/5 that customizable webhooks are not an outstanding feature for apps. You can do the same with a simple proxy.